jekyll/_config.yml

```diff
@@ -133,6 +140,13 @@ defaults:
     values:
       lang: "ja"
 
+  -
+    scope:
+      path: ""
+      type: "style"
+    values:
+      lang: "en"
+
```

Comment:
@chantelle11 This also needs to be put in the other configs file (`_config-stagging` and `_config-prod`)